invenio_files_rest/models.py

```diff
@@ -1307,26 +1307,20 @@ def delete(cls, bucket, key):
         return None
 
     @classmethod
-    def remove_many(cls, version_ids, bucket):
+    def remove_by_bucket(cls, bucket):
         """Permanently removes specific object versions from the database.
 
-        :param version_ids: list of version_id.
         :param bucket: The bucket (instance or id) to delete the object from.
         """
-        bucket = as_bucket(bucket)
         if bucket.locked:
             raise BucketLockedError()
-        object_versions = cls.query.filter(
-            cls.version_id.in_(version_ids), cls.bucket_id == bucket.id
+        object_versions = cls.get_by_bucket(
```

Comment:
```suggestion
        object_versions_query = cls.get_by_bucket(
```

It's getting difficult to understand what is a query and what is a list of objects, so I guess suffixing might be the way to clear this up.